checklists/alz_checklist.en.json

```diff
@@ -321,7 +322,7 @@
         {
             "category": "Identity and Access Management",
             "subcategory": "Identity",
-            "text": "If AD on Windows server in use, are the resources in Azure using the correct domain controller?",
+            "text": "If Microsoft Entra Domain Services on Windows server is in use, are the resources in Azure using the correct domain controller?",
```

Comment:
Could we change this to:

If domain controllers are being used, ensure that resources are set to use the correct domain controller.